exe/facter.cc

```diff
@@ -202,7 +206,12 @@ int main(int argc, char **argv)
         log_command_line(argc, argv);
 
         // Initialize Ruby in main
-        bool ruby = facter::ruby::initialize(vm.count("trace") == 1);
+        bool ruby;
+        if (vm.count("no-ruby")) {
+            ruby = false;
+        } else {
+            ruby = facter::ruby::initialize(vm.count("trace") == 1);
```

Comment:
Can these few lines be expressed as simply as `ruby = (vm.count("no-ruby") == 0) && facter::ruby::initialize(vm.count("trace") == 1);`?